test/30_debate/clientfunctions.js

```diff
@@ -279,7 +279,7 @@ describe("Debate client functions", () => {
     describe('suggestQuestion', () => {
         it('should accept a valid suggestion', (done) => {
             client.emit('suggestQuestion', 'This is my suggestion', res => {
-                res.should.equal(true);
+                res.should.not.equal(false);
```

Comment:
Same comment.

Reply:
suggestQuestion was modified to return false on error and the suggestion id on success (it is a necessary modification for the client app).
res.should.not.equal(false) allows me to verify that there is no error, rather than checking for a particular id.